dev/integration_tests/ios_platform_view_tests/ios/PlatformViewUITests/PlatformViewUITests.m

```diff
@@ -23,11 +23,18 @@ @interface PlatformViewUITests : XCTestCase
 @implementation PlatformViewUITests
 
 - (void)setUp {
+  [super setup];
   self.continueAfterFailure = NO;
 
   self.app = [[XCUIApplication alloc] init];
   [self.app launch];
 }
+
+- (void)tearDown {
+  [self.app terminate];
```

Comment:
Can you add a comment describing the bug you're trying to work around?  In theory it shouldn't be needed.